src/components/organisms/Navbar/Navbar/Navbar.tsx

```diff
@@ -276,6 +277,14 @@ const NavbarWrapper: React.FC<NavbarProps> = ({
   const overThreshold = useScrollThreshold(20);
   const [open, setOpen] = useState<boolean>(false);
 
+  useEffect(() => {
+    if (open) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'unset';
+    }
+  }, [open]);
```

Comment:
These kinda of things make me think that it would actually be easier to do the mobile navbar with the modal, and probably more accessible too. Probably outside of the scope of this PR though 😅